.github/actions/versioning/action.yml

```diff
@@ -1,12 +1,21 @@
 name: versioning
 description: Versioning
 inputs:
-  version_name:
-    description: Version name
+  RELEASE_TYPE:
+    description: Release Type
     required: false
+outputs:
+  version:
+    description: Version
 runs:
   using: composite
+
   steps:
-  - name: Tag
-    run: echo "${{ inputs.version_name }}-$(date -u +"%Y-%m-%dT%H:%M:%SZ")" > packages/core/dist/version
-    shell: bash
+    - name: Set Version
+      id: set_version
+      run: |
+        current_date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        version="${{ inputs.RELEASE_TYPE }}-GH_Run#:${{ github.run_number }}-Date:$current_date"
```

Comment:
<img width="901" alt="image" src="https://github.com/binary-com/deriv-app/assets/81898967/cc218e47-e90b-46f1-8f19-63eb22a524f7">
this would be the format of the slack notification.